lib/unicore/mktables

```diff
@@ -9657,12 +9334,7 @@ sub trace { return main::trace(@_) if main::DEBUG && $to_trace }
         return;
     }
 
-    sub set_fate {
-        my $self = shift;
-        my $fate = shift;
-        my $reason = shift;  # Ignored unless suppressing
```

Comment:
$reason is not optional, and the comment should be retained

Reply:
comment restored
nothing was enforcing `$reason` to be a mandatory argument, and it was possible before this change to call it without a reason, I can introduce a behavior change and make it mandatory

Reply:
`Too few arguments for subroutine 'Property::set_fate' at lib/unicore/mktables line 9176.`

Reply:
restoring it as an optional argument for now

Reply:
I thought I'd checked during my review that it was always present, but if not, ok